modules/framework/galasa-parent/dev.galasa.framework.api.ras/src/main/java/dev/galasa/framework/api/ras/internal/routes/RunQueryRoute.java

```diff
@@ -81,9 +82,11 @@ public class RunQueryRoute extends RunsRoute {
 	public static final String QUERY_PARAMETER_CURSOR = "cursor";
 	public static final String QUERY_PARAMETER_RUNNAME = "runname";
 	public static final String QUERY_PARAMETER_RUNID = "runid";
+	public static final String QUERY_PARAMATER_DETAIL = "detail";
+	public static final String DETAILS_METHOD_QUERY_PARAM_VALUE = "methods";
```

Comment:
Perhaps move the methods constant into the DetailsQueryParam class?

Reply:
Also, should we allow a detail value like`METHODS`,`Methods`, `meThOds`, etc (i.e. mixed case/upper case)?